client/src/main/java/org/schlunzis/kurtama/client/settings/Setting.java

```diff
@@ -10,6 +12,8 @@ public class Setting<T> {
     public static final Setting<String> HOST = new Setting<>("host", "127.0.0.1");
     public static final Setting<Integer> PORT = new Setting<>("port", 8007);
     public static final Setting<String> LANGUAGE = new Setting<>("language", Locale.US.toLanguageTag());
+    public static final Setting<String> SERVER_PATH = new Setting<>("server-path", System.getProperty("user.home") + File.separator + ".kurtama");
+    public static final Setting<ServerType> SERVER_TYPE = new Setting<>("server-type", ServerType.JAR);
```

Comment:
The lobby for the pure jar file server deployment seems to be quite big in this developer community, making it the default way. 😅